pkgs/build-support/docker/default.nix

```diff
@@ -922,6 +922,7 @@ rec {
                   --sort name \
                   --exclude=./proc \
                   --exclude=./sys \
+                  --exclude=.${builtins.storeDir} \
```

Comment:
Removing this again doesn't fail the test.

Reply:
I think this is ready to go; is there any other feedback you'd give me?

Reply:
The test fails and succeeds as it should now :+1: